app/views/application_records/show.haml

```diff
@@ -6,6 +6,10 @@ by:
 = @record.user.name_and_email
 = button_to 'Print this page',
   application_record_path(@record, format: :pdf), method: :get
+- if !@record.saved_for_later && @record.note_for_later.present?
```

Comment:
Yes, but there's a case where it was saved for later, but there was no note given.  In this case, `note_for_later` will always have been `nil`.